script/test

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -l
+
+if [ -f /.dockerenv ]; then
+    source ~/.bashrc
+    rvm use default
+    bundle config github.com $GITHUB_USER:$GITHUB_PASS
+    mv config/database{-jenkins,}.yml
+fi
+
+export RAILS_ENV=test
+export BUNDLE_WITHOUT=development:build
+
+exit_code=0
+
+function run {
+    declare -a tests_command=("$@")
+    echo ''
+    echo "=== Running \`${tests_command[*]}\`"
+    if ! ${tests_command[*]}; then
+        echo "=== These tests failed."
+        exit_code=1
+    fi
+}
+
+function info {
+    declare -a info_command=("$@")
+    echo ''
+    echo "=== Running for informational purposes \`${info_command[*]}\`"
+    if ! ${info_command[*]}; then
+        echo "== This test has errors and/or warnings. Please review results"
+    fi
+}
+
+run npm install -quiet
+run bundle install --quiet
+run bundle exec bowndler install -allow-root
+run bundle exec rake db:setup
+run bundle exec rubocop .
+run bundle exec cucumber -f progress
+run bundle exec rspec -f progress
+info ./node_modules/karma/bin/karma start spec/javascripts/karma.conf.js
+info brakeman -q --no-pager --ensure-latest
```

Comment:
IMHO, for fin_cap, you could add brakeman to "run" instead of adding "info". Wdyt?

Reply:
@tomas-stefano ye I agree. Most repo's have it set to "info", but considering theres no identified issues it should be "run". We just need to note tests will fail if Brakeman returns any errors/warnings. I've updated

Reply:
Yes, if Brakeman fail I will be expecting the code to not be merged at all.

Reply:
@tomas-stefano Unfortunately I have had to revert back to running Brakeman test as "info". I'm having issue with locales in docker. I've come across this before and I don't have an immediate fix.

`Error: invalid byte sequence in US-ASCII While processing /var/tmp/app/app/views/shared/_nav.html.erb`
`Location: /usr/local/rvm/gems/ruby-2.4.1/gems/brakeman-4.2.0/lib/brakeman/parsers/rails3_erubis.rb:78:in `

Reply:
Odd. Did you try it add the env lang as utf-8 `ENV LANG en_US.UTF-8`? There is a issue on debian (perhaps ubutu has the same problem as well) https://github.com/docker-library/ruby/issues/45

Reply:
Ye this is exactly what I was setting in the test script, which was not working. I have now set this directly in docker-compose which does the job :)

Reply:
Nice!!